pkg/services/testdata/noErrors.yml

```
@@ -124,4 +124,10 @@ workflows:
           dir: ./.circleci/scripts/
       - jobWithAWS_OIDC
       - ccc/random-job
-      - jobWithUnaccessibleOrb
+      - jobWithUnaccessibleOrb:
+          requires:
+              - jobWithAWS_OIDC: failed
+              - ccc/random-job:
+                  - success
+                  - canceled
+              - dummyJobWithStepParam: [canceled, failed]
```

Do we want to create a new test case here and not mix "unaccessible orb" with "job requires conditions"? E.g.

```suggestion
      - jobWithUnaccessibleOrb
      - jobWithComplexRequires:
          requires:
              - jobWithAWS_OIDC: failed
              - ccc/random-job:
                  - success
                  - canceled
              - dummyJobWithStepParam: [canceled, failed]
```

---

Good idea! I moved these test cases to a new file.